v3/src/components/data-display/models/data-configuration-model.ts

```diff
@@ -112,9 +108,30 @@ export const DataConfigurationModel = types
           : this.attributeDescriptions[role]
     },
     attributeID(role: GraphAttrRole) {
+
+      const defaultCaptionAttributeID = () => {
+        // We find the childmost collection and return the first attribute in that collection. If there is no
+        // childmost collection, we return the first attribute in the dataset.
+        const attrIDs = ['x', 'y', 'rightNumeric', 'topSplit', 'rightSplit']
```

Comment:
Not `legend` or `caption`? If so, you could use `TipAttrRoles`.

Reply:
Nope. Caption is not a candidate. But 'legend', so I'm adding that.